tests/cpp/test_multidevice_lower_communication.cpp

```diff
@@ -602,7 +603,7 @@ TEST_P(LowerCollectiveTest, ReduceScatterNoncontig) {
       executor_cache.runFusionWithInputs({in_tensor})[0].as<at::Tensor>();
 
   EXPECT_TRUE(out_tensor.t().is_contiguous());
-  EXPECT_TRUE(at::allclose(out_tensor, expected_output));
+  EXPECT_TRUE(at::equal(out_tensor, expected_output));
```

Comment:
Bumping the tolerance from `atol = 1e-8` to `atol = 1e-5` also works.
I chose `randint` since that is sufficient for the test, and may be more robust across number of devices and different runners.